iOS Example Frame/iOS Example Frame/CustomStyle2.swift

```diff
@@ -0,0 +1,239 @@
+    public var hint: ElementStyle? = nil
+    public var textfield: ElementTextFieldStyle = TextFieldCustom2()
+    public var error: ElementErrorViewStyle? = ErrorInputCustom2LabelStyle(text: "Enter County", textColor: Constants.errorLabelBackgroundColor)
+}
+
+struct BillingFormPostcodeCustom2CellStyle : CellTextFieldStyle {
+    public var isMandatory: Bool = true
+    public var isOptional: Bool = false
+    public var backgroundColor: UIColor = Constants.grayBackGroundColor
+    public var title: ElementStyle? = TitleLabelCustom2Style(text:  "Postcode*", textColor: Constants.fontColorLabel)
+    public var hint: ElementStyle? = nil
+    public var textfield: ElementTextFieldStyle = TextFieldCustom2()
+    public var error: ElementErrorViewStyle? = ErrorInputCustom2LabelStyle(text: "Enter Postcode", textColor: Constants.errorLabelBackgroundColor)
+}
+
+struct BillingFormCountryCustom2CellStyle: CellButtonStyle {
+    public var isMandatory: Bool = true
+    public var backgroundColor: UIColor = Constants.grayBackGroundColor
+    public var button: ElementButtonStyle = CountryCustom2FormButtonStyle()
+    public var isOptional: Bool = false
+    public var title: ElementStyle? = TitleLabelCustom2Style(text: "Country*", textColor: Constants.fontColorLabel)
+    public var hint: ElementStyle? = nil
+    public var error: ElementErrorViewStyle? = ErrorInputCustom2LabelStyle(text: "Enter Country", textColor: Constants.errorLabelBackgroundColor)
+}
+
+struct BillingFormPhoneNumberCustom2CellStyle : CellTextFieldStyle {
+    public var isMandatory: Bool = true
+    public var isOptional: Bool = false
+    public var backgroundColor: UIColor = Constants.grayBackGroundColor
+    public var title: ElementStyle? = TitleLabelCustom2Style(text:  "Phone*", textColor: Constants.fontColorLabel)
+    public var hint: ElementStyle? = HintInputCustom2LabelStyle(isHidden: true, text:  "")
+    public var textfield: ElementTextFieldStyle = TextFieldCustom2(isSupportingNumericKeyboard: true)
+    public var error: ElementErrorViewStyle? = ErrorInputCustom2LabelStyle(text:  "Enter Phone Number", textColor: Constants.errorLabelBackgroundColor)
+}
+
+struct BillingFormCustom2HeaderCellStyle: BillingFormHeaderCellStyle {
+    public var backgroundColor: UIColor = .clear
+    public var headerLabel: ElementStyle = HeaderCustom2LabelFormStyle()
+    public var cancelButton: ElementButtonStyle = CancelCustom2ButtonFormStyle()
+    public var doneButton: ElementButtonStyle = DoneCustom2FormButtonStyle()
+}
+
+struct HeaderCustom2LabelFormStyle: ElementStyle {
+    public var backgroundColor: UIColor = .clear
+    public var isHidden: Bool = false
+    public var text: String = "Billing Details"
+    public var font: UIFont = UIFont.systemFont(ofSize: 24.0)
+    public var textColor: UIColor  = .white
+}
+
+struct CancelCustom2ButtonFormStyle: ElementButtonStyle {
```

Comment:
All the objects in this file above this point are `internal` whilst their properties are `public`. Why all the objects below this point are `public` with `public` properties?

Reply:
I think even a `// MARK: Reason` above each group would help navigate the structure a little better

Reply:
public is removed